services/npm/npm.tester.js

```diff
@@ -231,9 +231,15 @@ t.create('when json is malformed for scoped package')
 t.create('types')
   .get('/types/chalk.json')
   .expectJSONTypes(
-    Joi.object().keys({ name: 'type definitions', value: isTypeDefinition })
+    Joi.object().keys({ name: 'types', value: isTypeDefinition })
   )
 
 t.create('no types')
```

Comment:
Do we still need this test (`/ypes/left-pad.json`)?

Reply:
Yea, I think it's important to have a test of a package with no types.

Reply:
But we have two tests named "no types" and only "link-into" does not have types. "left-pad" has types.

Reply:
Sorry, I was reading code from the wrong branch. 😝

left-pad uses `types`; chalk has `devDependencies`. I'll annotate what each of these are testing.